akka-docs/rst/java/remoting.rst

```diff
@@ -599,3 +599,11 @@ special configuration that sets both the logical and the bind pairs for remoting
       }
    }
   }
+
+Keep in mind that local.address must be in one of private network ranges: 
+
+* *10.0.0.0 - 10.255.255.255* (network class A)
+* *172.16.0.0 - 172.31.255.255* (network class B)
+* *192.168.0.0 - 192.168.255.255*(network class C)
+
+For further details see [RFC 1597](https://tools.ietf.org/html/rfc1597) and [RFC 1918](https://tools.ietf.org/html/rfc1918).
```

Comment:
hm yeah i guess that's one part of it; was more thinking about the docker side of things
since it really just depends which addresses docker actually sets up in its NAT - doesn't it?